styles.css

```diff
@@ -104,10 +108,72 @@ main h3 {
 	background-image: url("https://images.unsplash.com/photo-1617653695386-1d78957d33f8?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80");
 	background-size: cover;
 	background-repeat: no-repeat;
+	
+	color: white;
+	font-family: 'Montserrat', sans-serif;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.portfolio h2 {
+	color: white;
+	line-height: 1.6;
+	font-size: 4rem;
+	margin-bottom: 5%;
+}
+
+.proj {
+	width: 75%;
+	padding-bottom: 12%;	
+}
+
+.proj-img {
+	width: 30%;
+	height: 200px;
+}
+
+.proj-img img {
 	width: 100%;
-	height: 1000px;
 }
 
+.proj h3 {
+	padding-bottom: 2%;	
+}
+
+.proj-content {
+	display: flex;
+	align-items: center;
+	flex-direction: row;
+	
+}
+
+.proj-desc {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 200px;
+	width: 70%;
+}
+
+.proj-desc h4 {
+	line-height: 1.6;
+	font-size: 1rem;
+	margin-left: 18px;
+	text-align: left;
+}
+
+.proj-links {
+	text-align: right;
+
+}
+
+
```

Comment:
Unless you are going to be inserting something here, its worth removing the extra line breaks